tests/e2e/utils/CheReporter.ts

```diff
@@ -158,14 +153,12 @@ class CheReporter extends mocha.reporters.Spec {
 			// take screenshot and write to file
 			const screenshot: string = await this.driverHelper.getDriver().takeScreenshot();
 			const screenshotStream: WriteStream = fs.createWriteStream(screenshotFileName);
-			screenshotStream.write(Buffer.from(screenshot, 'base64'));
-			screenshotStream.end();
+			screenshotStream.write(Buffer.from(screenshot, 'base64'), (): WriteStream => screenshotStream.end());
 
 			// take page source and write to file
 			const pageSource: string = await this.driverHelper.getDriver().getPageSource();
 			const pageSourceStream: WriteStream = fs.createWriteStream(pageSourceFileName);
-			pageSourceStream.write(Buffer.from(pageSource));
-			pageSourceStream.end();
+			pageSourceStream.write(Buffer.from(pageSource), (): WriteStream => pageSourceStream.end());
```

Comment:
I am getting an error when I run `npm run test` because of `WriteStream` type:

```
Generating index.ts file...
utils/CheReporter.ts:156:81 - error TS2322: Type 'void' is not assignable to type 'WriteStream'.

156    screenshotStream.write(Buffer.from(screenshot, 'base64'), (): WriteStream => screenshotStream.end());
                                                                                    ~~~~~~~~~~~~~~~~~~~~~~

utils/CheReporter.ts:161:71 - error TS2322: Type 'void' is not assignable to type 'WriteStream'.

161    pageSourceStream.write(Buffer.from(pageSource), (): WriteStream => pageSourceStream.end());
                                                                          ~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: utils/CheReporter.ts:156
```

Reply:
indeed 
updated hasky pre-commit hook to prevent this kind of errors